pyenttec/__init__.py

```diff
@@ -209,10 +213,25 @@ def set_channel(self, chan, val):
             raise DMXAddressError("Channel index {} out of range. "
                                   "Universe size is {}."
                                   .format(chan, len(self.dmx_frame)))
+        except OverflowError:
+            raise DMXOverflowError("Channel value {} out of range. "
+                                  "DMX uses 8bit unsigned values (0-255)."
+                                  .format(chan, len(self.dmx_frame)))
```

Comment:
Looks like this format string only has one argument but two args have been passed to format, I'm assuming this is a copy/paste error and you meant to include both the channel and the value in the message.

Reply:
well spotted